compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -21,6 +21,20 @@ pub struct Lexer<'a> {
 
 pub type SpannedTokenResult = Result<SpannedToken, LexerErrorKind>;
 
+pub(crate) fn from_spanned_token_result(
+    x: &SpannedTokenResult,
```

Comment:
```suggestion
    token_result: &SpannedTokenResult,
```
Let's use clear naming when we can